src/main/java/org/dcsa/tnt/repository/EventSubscriptionRepository.java

```diff
@@ -15,10 +15,10 @@
 
 public interface EventSubscriptionRepository extends ExtendedRepository<EventSubscription, UUID> {
 
-    @Query("SELECT callback_url FROM event_subscription es"
+    @Query("SELECT es.* FROM event_subscription es"
             + "  JOIN event_subscription_event_types eset ON (es.subscription_id = eset.subscription_id)"
             + " WHERE eset.event_type = :eventType AND (:equipmentReference IS NULL or es.equipment_reference = :equipmentReference)")
-    Flux<String> findSubscriptionsByFilters(@Param("eventType") EventType eventType, @Param("equipmentReference") String equipmentReference);
+    Flux<EventSubscription> findSubscriptionsByFilters(@Param("eventType") EventType eventType, @Param("equipmentReference") String equipmentReference);
```

Comment:
Don't we have more filters than just equipomentReference and eventType?

Reply:
Possibly, but that is not a bug introduced by this commit. :)

One of my arguments for not changing this now is that I think we should "flip" the notification system from being an "on-submit" to a batch job and then this will have to be changed as a part of that.